app/views/summary.scala.html

```diff
@@ -15,177 +15,223 @@
  *@
 
 @import config.ApplicationConfig
-@import utils.ERSUtil
-@import utils.CountryCodes
+@import _root_.utils.ERSUtil
+@import _root_.utils.CountryCodes
+@import views.html.templates._
+@import views.html.govuk_wrapper
 
-@this(config: ApplicationConfig)
+@this(
```

Comment:
not sure where exactly this kicks in, but the hidden accessible text under File name repeats the text from the section above it, so it should be `Change File name`, but it actually is `Change Have options been granted, exercised, released (including exchanges), lapsed or cancelled during the tax year?`